rust/agama-network/src/model.rs

```diff
@@ -593,6 +606,9 @@ impl Default for Connection {
             ieee_8021x_config: Default::default(),
             autoconnect: true,
             state: Default::default(),
+            keep: true,
```

Comment:
master: Should we consider using `copy`? For me it sounds inconsistent having `keep` and `copy`.

Reply:
what about: **persistent** ... at the end it is about persist or not to disk using the **enum NMSettingsAddConnection2Flags
** or **enum NMSettingsUpdate2Flags**

Reply:
I mean, this is the connection one, so, for me having it a persist or persistent looks better while for the general state it is about copying or not copying the persistent configuration.